plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4598,6 +4615,27 @@ public void adjust(Mechanism mechanism) {
             state.update();
         }
 
+        // <--[mechanism]
+        // @object LocationTag
+        // @name waxed
+        // @input ElementTag(Boolean)
+        // @description
+        // Changes whether the sign at the location is waxed.
+        // @tags
+        // <LocationTag.waxed>
+        // -->
+        if (mechanism.matches("waxed") && mechanism.requireBoolean()) {
```

Comment:
Should use modern mechanism registration, `tagProcessor.registerMechanism` in the `register` method